tests/models/volunteers.test.js

```diff
@@ -0,0 +1,72 @@
+const chai = require('chai');
+const expect = chai.expect;
+var Department = require('../../models/volunteers').Department;
+var Volunteer = require('../../models/volunteers').Volunteer;
+var User = require('../../models/user').User;
+
+describe('Volunteer module db model', () => {
```

Comment:
tests don't reflect actual use of api, please test against real expected data, all these tests might pass with false positive results.